loopin.go

```diff
@@ -817,6 +839,9 @@ func (s *loopInSwap) processHtlcSpend(ctx context.Context,
 		// server cost balance.
 		s.cost.Server += htlcValue
 	} else {
+		// We needed another on chain tx to sweep the timeout clause,
+		// which we now include in our costs.
+		s.cost.Onchain += sweepFee
```

Comment:
Is it possible that we move this line into `publishTimeoutTx`? That way it could save some changes made.

Reply:
We call `publishTimeoutTx` every block, so we'd end up counting our fee on every republish. I thought just passing it out would be simpler than trying to add some once-only logic for storing fee, since we'd need to account for restarts as well.